package.json

```diff
@@ -110,5 +110,8 @@
     "setupFiles": [
       "./src/setupTests.js"
     ]
+  },
+  "dependencies": {
+    "@formatjs/intl": "^2.1.1"
```

Comment:
We was using `devDependencies` for all. Or you can move it to use `dependencies` for all.

Reply:
Fixed!